mta-otp-proxy/src/main/java/com/camsys/shims/gtfsrt/alerts/siri/deserializer/SiriDeserializerWithModifications.java

```diff
@@ -78,4 +79,27 @@ private void removeHtml(Siri siri) {
             }
         }
     }
+
+    private void filterLmm(Siri siri) {
+        //ServiceDelivery serviceDelivery = new ServiceDelivery();
+        //siri1.setServiceDelivery(serviceDelivery);
+        //siri1.setServiceDelivery(siri.getServiceDelivery());
+        Situations s1 = new Situations();
```

Comment:
Does this work?  You have the scoping of this very wrong.  You'll be setting the same bean across every seds.  This will result in only one GMS situationNumber across the entire feed, won't it?

If we name this `filteredSituations` instead of s1, this might be more obvious.  This initialization should occur along side the `sed.getSituations()`, no?

Reply:
I changed the `s1` to `filteredSitutations`.
I moved the initialization of the feed to right after `Situations s = seds.getSituations();`
It was working I think because from what I've seen, in the testing feed, `SituationExchangeDelivery` is always an array of one.